library/general/src/modules/Arch.rb

```diff
@@ -512,6 +512,22 @@ def is_wsl
       SCR.Read(path(".target.string"), "/proc/sys/kernel/osrelease").to_s.downcase.include?("-microsoft")
     end
 
+    # map the Arch.architecture to the arch expected by SCC
+    REG_ARCH = {
+      "s390_32" => "s390",
+      "s390_64" => "s390x",
+      # ppc64le is the only supported PPC arch, we do not have to distinguish the BE/LE variants
+      "ppc64"   => "ppc64le"
+    }.freeze
+    private_constant :REG_ARCH
+
+    # Returns the architecture expected by SCC
+    #
+    # @return [String] Architecture
+    def registration_arch
```

Comment:
I am just not sure about name. As it is not just registration, but also rpm architecture. so I would call it like rpm_arch as it can be used for other use cases not just for scc/registration.